.ruby-version

```diff
@@ -1 +1 @@
-system
+2.4.1
```

Comment:
@jmejia @icorson3 Is this file needed? We generated in the course of today's work (so maybe this was prompted by version changes from the am lesson setup?) but it has mostly caused issues, namely with rspec running.